src/views/dialogs/RateLimitDialog.tsx

```diff
@@ -0,0 +1,42 @@
+import styled, { AnyStyledComponent } from 'styled-components';
+
+import { STRING_KEYS } from '@/constants/localization';
+import { useStringGetter } from '@/hooks';
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { Dialog } from '@/components/Dialog';
+import { Icon, IconName } from '@/components/Icon';
+
+type ElementProps = {
+  preventClose?: boolean;
+  setIsOpen?: (open: boolean) => void;
+};
```

Comment:
nit / later code cleanup: thoughts on creating a type for this and share it among the dialogs?
or one for at least setIsOpen?